server/src/main/scala/za/co/absa/atum/server/api/database/runs/functions/CreateOrUpdateAdditionalData.scala

```diff
@@ -16,42 +16,29 @@
 
 package za.co.absa.atum.server.api.database.runs.functions
 
-import doobie.Fragment
 import doobie.implicits.toSqlInterpolator
-import doobie.util.Read
 import za.co.absa.atum.model.dto.AdditionalDataSubmitDTO
 import za.co.absa.atum.server.api.database.PostgresDatabaseProvider
 import za.co.absa.atum.server.api.database.runs.Runs
 import za.co.absa.atum.server.model.PartitioningForDB
-import za.co.absa.fadb.DBSchema
-import za.co.absa.fadb.doobie.DoobieFunction.DoobieSingleResultFunctionWithStatus
-import za.co.absa.fadb.doobie.{DoobieEngine, StatusWithData}
-import za.co.absa.fadb.status.handling.implementations.StandardStatusHandling
+import za.co.absa.db.fadb.DBSchema
+import za.co.absa.db.fadb.doobie.DoobieFunction.DoobieSingleResultFunctionWithStatus
+import za.co.absa.db.fadb.doobie.DoobieEngine
+import za.co.absa.db.fadb.status.handling.implementations.StandardStatusHandling
 import zio._
-import zio.interop.catz._
 import io.circe.syntax._
 
 import doobie.postgres.implicits._
-import doobie.postgres.circe.jsonb.implicits.jsonbPut
+import za.co.absa.db.fadb.doobie.postgres.circe.implicits.jsonbPut
```

Comment:
Yep. Let's use implicits from fa-db. Btw since the doobie-postgres-circe has been made a dependency of doobie's module in fa-db there is no need to explicitly build the project with that dependency.
In other words the below dependency is not needed anymore as it's brought in transitively by the fa-db's doobie module. Pls remove it.
lazy val pgCirceDoobie = "org.tpolecat" %% "doobie-postgres-circe" % "1.0.0-RC2"